src/components/Avatar.tsx

```diff
@@ -62,6 +65,7 @@ function Avatar({
     fallbackIconTestID = '',
     type = CONST.ICON_TYPE_AVATAR,
     name = '',
+    displayInDefaultIconColor = true,
```

Comment:
Since `displayInDefaultIconColor` is `true` by default and there is no instance of `Avatar` where we set it to `false`, we effectively set `fill` to **always** be `undefined`. This doesn't seem right